notebooks/official/experiments/get_started_with_vertex_experiments.ipynb

```diff
@@ -0,0 +1,1581 @@
+{
```

Comment:
<p>The above works but I'd suggest to use a setup script next time to make the package assembly a lot clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
reviewed